Client-Side/shecodesprogramming/src/Components/NavBar/navbar.css

```diff
@@ -0,0 +1,108 @@
+.livebooks{
+    height: 55px;
+    width: 55px;
+    /* margin-top: 20px; */
```

Comment:
Please remove commented code.

Reply:
I removed unwanted code could you please review it.